### PR TITLE
Unit tests for `file go`

### LINKED ISSUE #93: Unit tests for `file.go` module
Acceptance criteria:
- at least 50% code coverage

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Unit tests for `file go`

Fixes #93 

## Type of change

- Unit tests (no changes in the code)

## Testing steps

Done on CI

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: file_test.go [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+	defer mustRemoveTempDirectory(t, directory)
+
+	filename := directory + "tables.csv"
+	tableNames := []main.TableName{}
+
+	// just to be sure
+	assert.NoFileExists(t, filename, "File must not exist")
+
+	err := main.StoreTableNamesIntoFile(filename, tableNames)
+	assert.Nil(t, err, "Error should not be thrown for regular file name")
```
Comment (juandspy) [inline file_test.go:89]: I'd use `assert.NoError` instead of `assert.Nil`.

**Review by juandspy — APPROVED** [review]

File: file_test.go (hunk 1 above)
Comment (tisnik) [reply file_test.go:89]: true, lemme change that

Comment (tisnik) [reply file_test.go]: Fixed

**Comment by juandspy** [comment]
ok

**Review by Bee-lee — APPROVED** [review]

**Review by dpensi — APPROVED** [review]

**Review by dpensi — APPROVED** [review]

**Pull request merged**